tutorials/tut0.rst

```diff
@@ -159,11 +159,11 @@ Now quit out of psql; it's time to start the API!
 Step 5. Run PostgREST
 ---------------------
 
-PostgREST uses a configuration file to tell it how to connect to the database. Create a file :code:`tutorial.conf` with this inside:
+PostgREST uses a configuration file to tell it how to connect to the database. Create a file :code:`tutorial.conf` with this inside (if you installed via homebrew, the default port postgresql is listening on will be 5432 and not 5433):
 
 .. code-block:: ini
 
-  db-uri = "postgres://authenticator:mysecretpassword@localhost:5433/postgres"
+  db-uri = "postgres://authenticator:mysecretpassword@localhost:5433/tutorial"
```

Comment:
Hey there.

The `tutorial` db is not created in previous steps of the tutorial right? So it seems this won't work.

I think that for `tutorial` to be created we'd need to change the `POSTGRES_USER` env var(according to https://hub.docker.com/_/postgres).

Reply:
Oh interesting. I had assumed the docker image had a tutorial database in it so that's what I created when setting it up via homebrew. If that's not the case, then my bad, and please ignore! :)